config.go

```diff
@@ -62,6 +63,7 @@ type Config struct {
 	Table        string   `toml:"table"`
 	LogLevel     string   `toml:"log-level"`
 	Watch        bool     `toml:"watch"`
+	KubeConfig   string   `toml:"kubeconfig"`
```

Comment:
I mean that in our API I believe you have the option of specifying either a cube config file OR the API endpoint/key/cert files.  So just wondered if we needed that here too.